cli/src/commands/log.rs

```diff
@@ -116,6 +116,9 @@ pub(crate) struct LogArgs {
     /// Show patch
     #[arg(long, short = 'p')]
     patch: bool,
+    /// Print the number of commits instead of showing them
+    #[arg(long)]
```

Comment:
I think it's probably best to make this `conflicts_with_all` most of the other options for now. We can relax it later if it's a common request.